feature/mygroup/src/main/java/com/boostcamp/mapisode/mygroup/screen/GroupCreationScreen.kt

```diff
@@ -166,14 +216,17 @@ fun GroupCreationField(
 							.sizeIn(maxWidth = 380.dp, maxHeight = 380.dp)
 							.fillMaxWidth()
 							.aspectRatio(1f),
-						onClick = { },
-						showImage = false,
+						onClick = { onPhotoPickerClick() },
+						showImage = profileUrl == "",
```

Comment:
```suggestion
						showImage = profileUrl.isEmpty(),
```
제공하는 메서드를 사용하는 것도 좋을 것 같아요!